Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -895,7 +895,7 @@ private bool DownloadAccount(string account)
         /// </summary>
         public override void Disconnect()
         {
-            _client.ClientSocket.eDisconnect();
```

Comment:
Under what conditions can the client be null? Just curious, it might be indicative of another issue.

Reply:
It can be null if there is an unhandled IBAutomater error, e.g. if you run current master branch locally with an invalid `"ib-version"` or `"ib-tws-dir"` (unhandled Java exception, which is fixed in this IBAutomater update):

```
,[ERROR] FATAL UNHANDLED EXCEPTION: System.NullReferenceException: Object reference not set to an instance of an object,
  at QuantConnect.Brokerages.InteractiveBrokers.InteractiveBrokersBrokerage.Disconnect () [0x00000] in :0 ,
  at QuantConnect.Brokerages.InteractiveBrokers.InteractiveBrokersBrokerage.OnIbAutomaterExited (System.Object sender, QuantConnect.IBAutomater.ExitedEventArgs e) [0x0004c] in :0 ,
  at QuantConnect.IBAutomater.IBAutomater+c__DisplayClass30_0.b__2 (System.Object sender, System.EventArgs e) [0x00028] in :0 ,
  at System.Diagnostics.Process.OnExited () [0x0003f] in :0 ,  at System.Diagnostics.Process.RaiseOnExited () [0x0002c] in :0 ,
  at System.Diagnostics.Process.CompletionCallback (System.Object context, System.Boolean wasSignaled) [0x00006] in :0 ,
  at System.Threading.RegisteredWaitHandle.DoCallBack (System.Object timedOut) [0x00008] in :0 ,
  at System.Threading.QueueUserWorkItemCallback.WaitCallback_Context (System.Object state) [0x00007] in :0 ,
  at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00071] in :0 ,
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in :0 ,
  at System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem () [0x00021] in :0 ,  at System.Threading.ThreadPoolWorkQueue.Dispatch () [0x00074] in :0 ,
  at System.Threading._ThreadPoolWaitCallback.PerformWaitCallback () [0x00000] in :0 
```